models/Status.yaml

```diff
@@ -175,3 +175,6 @@ properties:
   limit_version:
     type: integer
     description: /limit endpoint current version
+  uptime:
+    type: integer
```

Comment:
we don't make the difference here between `integer`, `long`, `unsigned` numbers ? `integer` means `number` ? Or current entries are incorrect and should all be `number` instead ?

Reply:
Think integer is correct for the OpenAPI spec.